### DISCUSSION TITLE
How to improve bad segmentation?

### DISCUSSION THREAD

**Discussion by ryokisasaki** [simnibs/simnibs#380] [7.Creating head models]
Dear simnibs community,

I have run a MRI segmentation using the following code in the simnibs prompt window: 
cd /d I:\tDCS project\"ID"
charm enrie 1.nii --forceqform

After this process, I checked a charm report and found the bad segmentstion (please see the attached picture).
I visited the simnibs website (https://simnibs.github.io/simnibs/build/html/tutorial/advanced/fix_affine_registration.html), and I knew manual edition is needed using the '--usesettings argument'. I chenged the default value of affine scales in the settings. However, I do not know how to re-run this process using the usesettings argument in the prompt window. I would greatly appreciated if you could provide the code based on the above code. 

![bad segmentation](https://github.com/simnibs/simnibs/assets/173213578/910ef123-4487-43a8-a82d-f83a273d2fae)

Best regards,
Ryoki

**Comment by oulap** [comment]
Hi, 

Once you have the edited settings file, e.g., my_settings.ini, you can call charm again like this:

`charm subject_id t1.nii --forceqform --usesettings my_settings.ini`

Note that in this case you should run the charm call from the location where the `my_settings.ini` file is. Otherwise you need to pass the full path to the ini-file.

**Comment by ryokisasaki** [comment]
Thank you very much for providing the code and suggestions. I used this code, and it run without any problems.

Many thanks,
Ryoki
